app/renderer/actions/Session.js

```diff
@@ -588,19 +610,16 @@ export function getRunningSessions () {
     dispatch({type: GET_SESSIONS_REQUESTED});
     if (avoidServerTypes.includes(serverType)) {
       dispatch({type: GET_SESSIONS_DONE});
-    } else {
-      ipcRenderer.send('appium-client-get-sessions', {
-        host: serverInfo.hostname, port: serverInfo.port, path: serverInfo.path, ssl: serverInfo.ssl
-      });
-      ipcRenderer.once('appium-client-get-sessions-response', (evt, e) => {
-        const res = JSON.parse(e.res);
-        dispatch({type: GET_SESSIONS_DONE, sessions: res.value});
-        removeRunningSessionsListeners();
-      });
-      ipcRenderer.once('appium-client-get-sessions-fail', () => {
-        dispatch({type: GET_SESSIONS_DONE});
-        removeRunningSessionsListeners();
-      });
+      return;
+    }
+
+    const {hostname, port, path, ssl} = serverInfo;
+    try {
+      const adjPath = path.endsWith('/') ? path : `${path}/`;
+      const res = await ky(`http${ssl ? 's' : ''}://${hostname}:${port}${adjPath}sessions`).json();
+      dispatch({type: GET_SESSIONS_DONE, sessions: res.value});
+    } catch (ign) {
+      dispatch({type: GET_SESSIONS_DONE});
```

Comment:
should we completely silence the error here?

Reply:
I guess we could log it